spec/controllers/users_controller_spec.rb

```diff
@@ -44,6 +46,32 @@
         expect(assigns(:user)).to be_instance_of(User)
       end
     end
+    
+    context "sending emails" do
+      context "valid input details" do
+        
+        before { post :create, user: { email_address: 'knoxjeffrey@outlook.com', password: 'password', full_name: 'Jeff Knox' } }
+        after { ActionMailer::Base.deliveries.clear }
+        
+        it "sends out the email" do
+          expect(ActionMailer::Base.deliveries.last.to).to eq(['knoxjeffrey@outlook.com'])
+        end
+      
+        it "sends email containing the users full name" do
+          expect(ActionMailer::Base.deliveries.last.body).to include("Jeff Knox")
+        end
```

Comment:
I would probably add also a test whether the email message includes the callback link to your app, because that is the key part of whole process.